include/mbedtls/ssl.h

```diff
@@ -4360,6 +4360,24 @@ void mbedtls_ssl_conf_cert_req_ca_list(mbedtls_ssl_config *conf,
  *                 with \c mbedtls_ssl_read()), not handshake messages.
  *                 With DTLS, this affects both ApplicationData and handshake.
  *
+ * \note           Defragmentation of incoming handshake messages in TLS
+ *                 (excluding DTLS) is supported with some limitations:
```

Comment:
I find the wording potentially confusing - one could get the impression that defragmentation is not supported for DTLS. Suggesting "Defragmentation of incoming handshake messages is supported in both DTLS and TLS but has some limitations in TLS"